src/templates/Footer/index.js

```diff
@@ -82,8 +82,9 @@ function renderPhoneInfo(phoneNumber, hours) {
 
       <div className={styles['hours']}>
         {
-          hours && hours.map(hour =>
+          hours && hours.map((hour, i) =>
             <Text
+              key={`hours-${i}`}
```

Comment:
It's bad practice, I believe, to use the map index as a key. Just use `hour`?

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Reply:
It doesn't really matter for a component that will never get updated but OK FINE

Reply:
haha

Reply:
I'm surprised we don't have this linter rule in place.